gluster-block.spec.in

```diff
@@ -52,6 +52,7 @@ rm -rf ${RPM_BUILD_ROOT}
 %{_sbindir}/gluster-block
 %{_sbindir}/gluster-blockd
 %{_mandir}/man8/gluster-block*.8*
+%dir %{_localstatedir}/log/gluster-block
```

Comment:
@obnoxxx do you think, this patch still add some value in the daemon not started case ?